src/libraries/Microsoft.PowerFx.Core/IR/Nodes/TextLiteralNode.cs

```diff
@@ -20,7 +20,7 @@ public TextLiteralNode(IRContext irContext, string value)
 
         public override string ToString()
         {
-            return $"Text({LiteralValue})";
+            return $"\"{LiteralValue}\":s";
```

Comment:
>LiteralValue [](http://example.com/codeflow?start=24&length=12)

Is the result ok if that string contains double quotes?

Reply:
Good call.  I changed it to escape the string, as is done for ToExpression string values.